### ISSUE TITLE
[question] GP model underfitting

### ISSUE THREAD

**Issue by rgreen1995** [cornellius-gp/gpytorch#697]
Hi 
I'm trying to fit a model with 9 variable and 1 output, the size of the data is  O(10**4). The model seems to be underfitting quite a lot . The variance of the predictions is much smaller than the true value, this suggests to me that the length scale is too large ? 

![image](https://user-images.githubusercontent.com/28905576/57694684-46d92380-7644-11e9-8a39-66c32da9c026.png)

To create the model I use the following code: 

    class GPRegressionModel(gpytorch.models.ExactGP):
        def __init__(self, train_x, train_y, likelihood):
            super(GPRegressionModel, self).__init__(train_x, train_y, likelihood)
            self.mean_module = ConstantMean()
            self.base_covar_module = ScaleKernel(MaternKernel(nu =0.5 ,ard_num_dims = 9))
            self.covar_module = InducingPointKernel(self.base_covar_module,
                                                inducing_points=train_x[:1000, :],
                                                likelihood=likelihood)

    def forward(self, x):
        mean_x = self.mean_module(x)
        covar_x = self.covar_module(x)
        return MultivariateNormal(mean_x, covar_x)

    # Find optimal model hyperparameters
    model.train()
    likelihood.train()

    # Use the adam optimizer
    optimizer = torch.optim.Adam(model.parameters(), lr=0.1)

    # "Loss" for GPs - the marginal log likelihood
    mll = gpytorch.mlls.ExactMarginalLogLikelihood(likelihood, model)

    training_iterations = 50
    def train():
        for i in range(training_iterations):
           # Zero backprop gradients
            optimizer.zero_grad()
            # Get output from model
           output = model(train_x)
            # Calc loss and backprop derivatives
            loss = -mll(output, train_y)
            loss.backward()
            if i % 10 == 0 :
              print('Iter %d/%d - Loss: %.3f' % (i + 1, training_iterations, loss.item()))
            optimizer.step()
            torch.cuda.empty_cache()

    # See dkl_mnist.ipynb for explanation of this flag
    with gpytorch.settings.use_toeplitz(True):
        %time train()

Any help on this would be greatly appreciated
